packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.tsx

```diff
@@ -21,11 +21,21 @@ type RiskManagementItemProps = {
     modal_body_content: React.ReactNode;
     is_deal_cancellation?: boolean;
     value?: number | null;
-    type?: string;
+    validation_params?: {
+        [key: string]: { min: number; max: number };
+    };
+    type?: 'take_profit' | 'stop_loss';
 };
 
 const RiskManagementItem = observer(
-    ({ label, modal_body_content, is_deal_cancellation = false, value, type }: RiskManagementItemProps) => {
+    ({
+        label,
+        modal_body_content,
+        is_deal_cancellation = false,
+        value,
+        validation_params,
```

Comment:
Not sure,, will it be better or not, but you might not pass `validation_params` props, because they are already included in `contract_info` in line 43

Reply:
Can extract it right in this component, without props drilling

Reply:
Ah nice, I haven't realized this

Reply:
Thanks, I applied this suggestion. Not sure how I missed it in the first place, the component already has contract_info.